datafusion/optimizer/src/eliminate_duplicated_expr.rs

```diff
@@ -20,10 +20,9 @@
 use crate::optimizer::ApplyOrder;
 use crate::{OptimizerConfig, OptimizerRule};
 use datafusion_common::tree_node::Transformed;
-use datafusion_common::Result;
+use datafusion_common::{IndexSet, Result};
 use datafusion_expr::logical_plan::LogicalPlan;
 use datafusion_expr::{Aggregate, Expr, Sort, SortExpr};
-use indexmap::IndexSet;
```

Comment:
is `indexmap` also removed from `datafusion/optimizer/Cargo.toml`?

Reply:
I think it cannot be removed because of `indexmap::Equivalent`.

https://github.com/apache/datafusion/blob/0c01c0e688037d855a87d70b6b7025d91d62c077/datafusion/optimizer/src/join_key_set.rs#L151-L155